src/systems/userinput/devices/daydream-controller.js

```diff
@@ -1,5 +1,8 @@
 import { paths } from "../paths";
 import { Pose } from "../pose";
+import { applyArmModel } from "../arm-model.js";
+
+const ONES = new THREE.Vector3(1, 1, 1);
```

Comment:
If only `THREE.Vector3.ONE` was a thing, but then again I guarantee we would get clobbered at some point and cause all sorts of bugs, so I take it back.